modules/ocm-nw-networkpolicy-create.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/network_policy/creating-network-policy.adoc
+// * post_installation_configuration/network-configuration.adoc
+
+:name: network
+:role: admin
+ifeval::["{context}" == "configuring-multi-network-policy"]
```

Comment:
Since the procedure doesn't seem to rely on these, and it's for network policy on secondary interfaces anyway, you can omit this from all newly created files.